File: Legião.java
public class Legião {
   	// instance variables
	public String callSign;
	private int shieldStrength;

	// methods
	public String fireMissile() {
		return "Pew!";
	}

	public void reduceShield(int amount) {
		shieldStrength -= amount;
	}
} 

File: Legião.swift
public Legião: NSObject {
	
	// instance variables
	public var callSign: String
	private var _shieldStrength: Int

	// methods
	func fireMissile() -> String {
		return "Pew!"
	}

	func reduceShield(amount: Int) {
		_shieldStrength -= amount
	}
}